tests/test_utils.py

```diff
@@ -13,3 +14,53 @@ def test_extract_email_from_globus_jwt_malformed(identity_jwt):
     malformed_jwt = ".".join(jwt_segments)
     with pytest.raises(Exception):
         extract_email_from_globus_jwt(malformed_jwt)
+
+
+notebook_with_outputs = {
+    "cells": [
+        {
+            "cell_type": "code",
+            "execution_count": 1,
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {"text/plain": ["Hello"]},
+                    "execution_count": 1,
+                    "metadata": {},
+                    "output_type": "execute_result",
+                }
+            ],
+            "source": ["print('Hello')"],
+        }
+    ],
+    "metadata": {},
+    "nbformat": 4,
+    "nbformat_minor": 4,
+}
+cleared_notebook = {
+    "cells": [
+        {
+            "cell_type": "code",
+            "execution_count": 1,
+            "metadata": {},
+            "outputs": [],
+            "source": ["print('Hello')"],
+        }
+    ],
+    "metadata": {},
+    "nbformat": 4,
+    "nbformat_minor": 4,
+}
+
+
+def test_clear_cells():
+    assert clear_cells(notebook_with_outputs) == cleared_notebook
+
+
+def test_is_over_size_limit():
+    assert not is_over_size_limit(cleared_notebook)
+
+    # 6MB worth of "a"s
```

Comment:
the kinda notebook written by the monkey with a typewriter before he works up to shakespeare

Reply:
or maybe the monkey has transcended shakespeare, gone back to their roots